src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size;
+    private T[] arrList;
+    private T value;
+
+    public ArrayList() {
+        arrList = (T[]) new Object[DEFAULT_CAPACITY];
+        size = 0;
+    }
+
+    public void isIndexOutOfBounds(int index) {
```

Comment:
```suggestion
    private void checkIndex(int index) {
```
and put private after public ones